client/data/hosting/use-cache.ts

```diff
@@ -88,42 +88,45 @@ export const useSetEdgeCacheMutation = (
 			queryClient.setQueryData( [ USE_EDGE_CACHE_QUERY_KEY, siteId ], active );
 			return previousData;
 		},
-		onSuccess( ...args ) {
-			const [ , { active } ] = args;
```

Comment:
This change obviously isn't necessary, but I chose to do it for a few reasons:
1. It's more readable.
2. This is consistent with my new `onSuccess` implementation in `useEdgeCacheDefensiveModeMutation`.
3. We have type checking for `options.onSuccess`, so Typescript will complain if we pass the wrong or too few arguments.